web/server/entrypoint_test.go

```diff
@@ -172,8 +174,23 @@ func TestShutdown(t *testing.T) {
 			// Asserting not nil here to ensure process is dead
 			test.That(tb, rdkStatus, test.ShouldNotBeNil)
 		})
-		test.That(t, (err == nil || err.Error() == `rpc error: code = DeadlineExceeded
-			desc = context deadline exceeded`), test.ShouldBeTrue)
+		test.That(t, isExpectedShutdownError(err, testLogger), test.ShouldBeTrue)
 		test.That(t, serverLogObserver.FilterLevelExact(zapcore.ErrorLevel).Len(), test.ShouldEqual, 0)
 	})
 }
+
+func isExpectedShutdownError(err error, testLogger logging.Logger) bool {
+	expectedErrorCode := map[codes.Code]bool{
+		codes.Unknown:          true,
+		codes.Unavailable:      true,
+		codes.DeadlineExceeded: true,
+		codes.Internal:         true,
+	}
```

Comment:
I thought we were swallowing `Unknown` and `Internal`; could either of those be returned from `Shutdown` through a robot client?

Reply:
oops you're right, no need to check for these